catalog-info.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
       title: Team Slack
       icon: chat
 spec:
+  system: App Management | Developer Metrics | Marketplace | Sidepanel | Support Documentation
```

Comment:
@claudioprocore what System does blueprinter fall under? I added all of the team-dmx systems here.